.github/workflows/ci.yml

```diff
@@ -1074,10 +1074,10 @@ jobs:
       PR_LABELS: "${{needs.build-info.outputs.pull-request-labels}}"
       FULL_TESTS_NEEDED: "${{needs.build-info.outputs.full-tests-needed}}"
       DEBUG_RESOURCES: "${{needs.build-info.outputs.debug-resources}}"
-      BACKEND: "postgres"
+      BACKEND: "mysql"
       PYTHON_MAJOR_MINOR_VERSION: "${{needs.build-info.outputs.default-python-version}}"
-      POSTGRES_VERSION: "${{needs.build-info.outputs.default-postgres-version}}"
-      BACKEND_VERSION: "${{needs.build-info.outputs.default-python-version}}"
+      MYSQL_VERSION: "${{steps.selective-checks.outputs.default-mysql-version}}"
```

Comment:
You cannot use "step.selective-checks" here, because there is no "selective-checks" step in this job. The "selective-checks" step is run in build-info job and the step output is passed-through to be the "build-info" job output, that's why we need to use "needs.build-info" here (as in the original job).

Reply:
(this is why the workflow is failing now I guess).

Reply:
Oh... I've just copy and paste from wrong place 🤦

Reply:
This is how those errors came here in the first place so :smile:

Reply:
This SO answers takes first place and SO questions second. 🤣